pySDC/implementations/sweeper_classes/Runge_Kutta.py

```diff
@@ -655,6 +655,71 @@ def get_update_order(cls):
         return 4
 
 
+class ESDIRK43(RungeKutta):
+    """
+    A-stable embedded RK pair of orders 4 and 3, ESDIRK4(3)6L[2]SA.
+    Taken from [here](https://ntrs.nasa.gov/citations/20160005923)
+    """
+
+    s2 = 2**0.5
+
+    nodes = np.array([0, 1 / 2, (2 - 2**0.5) / 4, 5 / 8, 26 / 25, 1.0])
+    matrix = np.zeros((6, 6))
+    matrix[1, :2] = [1 / 4, 1 / 4]  # OK
+    matrix[2, :3] = [
+        (1 - 2**0.5) / 8,
+        (1 - 2**0.5) / 8,
+        1 / 4,
+    ]  # OK
```

Comment:
what's with the `OK`?

Reply:
Some previous consistency checks, that have been removed.